lib/src/commonMain/kotlin/org/reduxkotlin/Definitions.kt

```diff
@@ -52,3 +58,26 @@ fun <State> middleware(dispatch: (Store<State>, next: Dispatcher, action: Any) -
             }
         }
     }
+
+
+/**
+ * Convenience function for creating a [ReducerForActionType]
+ * usage:
+ *   sealed class LoginScreenAction
+ *   data class LoginComplete(val user: User): LoginScreenAction()
+ *
+ *   val loginReducer = sealedReducer<AppState, LoginAction> { state, action ->
+ *       when(action) {
+ *           is LoginComplete -> state.copy(user = action.user)
+ *       }
+ *   }
+ */
+inline fun <TState, reified TAction> reducerForActionType(crossinline reducer: ReducerForActionType<TState, TAction>): Reducer<TState> {
+    return {state, action ->
```

Comment:
have you considered adding detekt/static analysis for CI builds?

https://github.com/arturbosch/detekt

```suggestion
    return { state, action ->
```